src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,166 @@
+            _item_handle = handle;
+        }
+
+        /// <summary>
+        /// Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Url
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_url_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Title of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Title
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Original Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string OriginalUrl
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_original_url_get(_item_handle);
+            }
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Back Forward list of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardList
+    {
+        private IntPtr _list_handle;
+
+        /// <summary>
+        /// Creates a Back Forward List object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal BackForwardList(IntPtr handle)
+        {
+            _list_handle = handle;
+        }
+
+        /// <summary>
+        /// Gets current item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetCurrentItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_current_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets previous item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetPreviousItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_previous_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets the back forward list count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public uint Count
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_count(_list_handle);
+            }
+        }
+        /// <summary>
+        /// Gets the list containing the items preceding the current item
+        /// limited by limit.
+        /// </summary>
+        /// <param limit="limit"> limit The number of items to retrieve.</param>
```

Comment:
Parameter 'limit' is missing the 'name' attribute in the documentation. It should have been `<param name="limit">`

Reply:
Done